website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -522,6 +572,12 @@ together transforms to create a pipeline that successively modifies input data:
               | [Third Transform])
 {{< /highlight >}}
 
+{{< highlight go >}}
+[Second PCollection] := beam.ParDo(s, [First Transform], [Initial Input PCollection])
+[Third PCollection] := beam.ParDo(s, [Second Transform], [Second PCollection])
```

Comment:
Suggested change:

```
[Third PCollection] := beam.ParDo(s, [Second Transform], [Second PCollection])
```

to

```
[Third PCollection] := beam.ParDo(scope, [Second Transform], [Second PCollection])
```

Reply:
This one is reasonable since it's abstract code anyway. However, I'd rather not force a generation of Go SDK users to repeatedly use `scope` for a pervasive variable. Eventually too much such repetition is not useful and draws focus from the distinct parts of the call.

I agree that towards the pipeline phase, we should explain the scope variable better, what it's for (referring to the section on composites), and that conventionally, it's abbreviated to 's' which is like how `context.Context` is abbreviated to `ctx`, and `io.Reader` and `io.Writer` become `r` and `w` respectively.

Reply:
I think the abbreviation makes sense when one is looking at the whole code.  The rendering on the online template breaks context.  It may be my own issues with reading when dealing with context shift so I'm probably not the best judge for the average audience.  Thank you for listening to the comment though.

Reply:
Re-commenting, since this is done.